study-builder/studies/testboston/i18n/en.conf

```diff
@@ -168,6 +168,36 @@
+    "neg_item4": "You may also always contact the team at <a class=\"Link Link_semi-bold\" href=\"tel:(617) 525-4220\">(617) 525-4220</a> or at <a class=\"Link Link_semi-bold\" href=\"mailto:info@testboston.org\">info@testboston.org</a> for additional information.",
+    "more_info": "Where can I get more information about COVID-19?",
+    "more_info_pos": "<strong>Easy-to-read information with pictures:</strong> <a href=\"https://www.healthwise.net/partnersmychart/Content/StdDocument.aspx?DOCHWID=custom.partners.covidfamily\" target=\"_blank\">Caring for Yourself or a Family Member with COVID-19</a>.",
+    "more_info_item1": "<strong>Massachusetts residents:</strong> The Massachusetts Department of Public Health (DPH) Hotline can be reached at <a class=\"Link Link_semi-bold\" href=\"tel:(617) 983-6800\">(617) 983-6800</a> or at <a class=\"Link Link_semi-bold\" href=\"tel:211\">211</a>, 24 hours a day, 7 days per week. <strong>TTY/TDD</strong> <a class=\"Link Link_semi-bold\" href=\"tel:(617) 624-6001\">(617) 624-6001</a>. Also check the website: <a href=\"https://www.mass.gov/resource/information-on-the-outbreak-of-coronavirus-disease-2019-covid-19\" target=\"_blank\">Massachusetts DPH</a>.",
+    "more_info_item2_pos": "<strong>Boston residents:</strong> Find reliable information including information on free testing at the Boston Public Health Commission, <a class=\"Link Link_semi-bold\" href=\"tel:(617) 534-5395\">(617) 534-5395</a>. Also check the website: <a href=\"https://www.bphc.org/onlinenewsroom/Blog/Lists/Posts/Post.aspx?List=24ee0d58%2D2a85%2D4a4a%2D855b%2Df5af9d781627&ID=1282&RootFolder=%2Fonlinenewsroom%2FBlog%2FLists%2FPosts&Source=https%3A%2F%2Fwww%2Ebphc%2Eorg%2FPages%2Fdefault%2Easpx&Web=03126e14%2D4972%2D4333%2Db8a3%2D800cbc1cafce\" target=\"_blank\">BPHC</a>.",
+    "more_info_item2_neg": "<strong>Boston residents:</strong> Find reliable information at the Boston Public Health Commission, <a class=\"Link Link_semi-bold\" href=\"tel:(617) 534-5395\">(617) 534-5395</a>. Also check the website: <a href=\"https://www.bphc.org/onlinenewsroom/Blog/Lists/Posts/Post.aspx?List=24ee0d58%2D2a85%2D4a4a%2D855b%2Df5af9d781627&ID=1282&RootFolder=%2Fonlinenewsroom%2FBlog%2FLists%2FPosts&Source=https%3A%2F%2Fwww%2Ebphc%2Eorg%2FPages%2Fdefault%2Easpx&Web=03126e14%2D4972%2D4333%2Db8a3%2D800cbc1cafce\" target=\"_blank\">BPHC</a>.",
+    "other_item": "This result means we are unable to get a result from your nasal swab. Some reasons this may occur include:",
+    "other_reason1": "A problem with the sample received. This can occur if the swab was not in the tube, the swab was found upside down or if the tube was not labelled correctly.",
+    "other_reason2": "The swab did not arrive in the lab within 72 hours of being collected.",
```

Comment:
As per zims comment: https://broadinstitute.slack.com/archives/G012Y2S45KN/p1599748272417200 , we need this text changed to "The sample could not be processed within 72 hours of collection"

Reply:
In another Slack channel with the PIs, the PIs indicated that we should keep it for now.